app.config.js

```diff
@@ -4,11 +4,17 @@ module.exports = {
   // networks in their wallet.
   // Ocean Protocol contracts are deployed for: 'mainnet', 'rinkeby', 'development'
   network: process.env.GATSBY_NETWORK || 'mainnet',
+
+  // List of supported networkId the market works with. Used for showing UI hints.
+  supportedNetworks: [1, 3, 4, 137, 80001],
```

Comment:
do we need this? we already have a list of supported networks [here](https://github.com/oceanprotocol/ocean.js/blob/main/src/utils/ConfigHelper.ts) so something like `ConfigHelper().getConfig()` returning `null` means we don't support it. I want to prevent having multiple lists with the same meaning.

Reply:
great point, will remove this and rather cross-reference with `ConfigHelper`. Btw, `80001` is the Matic Mumbai test network but I'm pretty sure we did not deploy contracts to it. But we maybe should before release and then add it to `ConfigHelper`